src/main/java/uk/gov/companieshouse/api/accounts/model/ixbrl/company/Company.java

```diff
@@ -24,4 +34,20 @@ public String getCompanyName() {
     public void setCompanyName(String companyName) {
         this.companyName = companyName;
     }
+    
+    private void getCompanyJurisdiction(String companyNumber) {
+        String companyPrefix = companyNumber.substring(0,2);
```

Comment:
This will throw an error if company number is blank. Is that expected or should this be within a null/blank check?